### DISCUSSION TITLE
list by related_name is empty

### DISCUSSION THREAD

**Discussion by NikitaAleshnikov** [collerek/ormar#218] [Q&A] [answered]
Models:
![image](https://user-images.githubusercontent.com/78476951/120495852-be15b780-c3c5-11eb-8143-7f8cc4281138.png)
![image](https://user-images.githubusercontent.com/78476951/120495948-d2f24b00-c3c5-11eb-9c45-2c940bfe2d05.png)
in DB:
![image](https://user-images.githubusercontent.com/78476951/120496128-fcab7200-c3c5-11eb-8ebe-0371f0bcbc35.png)
![image](https://user-images.githubusercontent.com/78476951/120496185-09c86100-c3c6-11eb-800e-afccfbac1533.png)
if i try to get reverse FK, chat.contests return [ ]
what is wrong?

**Comment by collerek** [comment, marked as answer]
It's hard to tell without the actual code you use but i would guess you do not provide `select_related` or `prefetch_related`. If you query from chat the related models are not loaded by default. If you query from contest the models are not loaded by default too but you get a primary key only model (special model with only on populated that you can load in separate call)

So you need to run
```python
chat = await Chat.objects.select_related("contests").get(id=xxx)
assert len(chat.contests)>0

#or you can use separate query if you already have chat
await chat.contests.all()
```

**Reply by NikitaAleshnikov** [reply]
Thanks!
